lib/dialects/mysql/query-generator.js

```diff
@@ -65,6 +66,7 @@ const QueryGenerator = {
       engine: options.engine,
       charset: (options.charset ? ' DEFAULT CHARSET=' + options.charset : ''),
       collation: (options.collate ? ' COLLATE ' + options.collate : ''),
+      format: options.format ? ' ROW_FORMAT=' + options.format : '',
```

Comment:
`format` name is quite common and can be misunderstood, perhaps name it `rowFormat`